lessons/Projects/project-threats/langs/en-us/pages/rubric-threats-to-validity.adoc

```diff
@@ -0,0 +1,48 @@
+= Threats to Validity Rubric
+
+[cols="2,4,4,4", options="header"]
+|===
+|
+| Wow!
+| Getting There
+| Needs Improvement
+
+| Statistical question
```

Comment:
I think I need to hear move about why the question itself belongs here. Somehow I'm just not able to wrap my head around choosing a hi-threat v. lo-threat question! But obviously you do, so I want to hear it from you.

Reply:
nah, I agree with you. It can go!